src/ui/components/Icon/styles.scss

```diff
@@ -37,3 +38,26 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') center no-repeat;
+  background-size: cover;
+  height: 20px;
+  width: 20px;
+}
+
+.Icon-themes {
+  background: url('./themes.svg') center no-repeat;
+  background-size: cover;
+  height: 50px;
+  min-width: 41.5px;
```

Comment:
The extensions icon is 139x134 and the themes icon is 112x135. I think generally the values we get from the mocks are 3x bigger than what we want and those are pretty closed to what these are so I just divided by 3 in each dimension. The sizes are slightly smaller now.